internal/uvm/create.go

```diff
@@ -318,6 +318,10 @@ func Create(opts *UVMOptions) (_ *UtilityVM, err error) {
 			}
 		}
 
+		if os.Getenv("HCSSHIM_UVM_CONSOLEPIPE") != "" {
```

Comment:
I'd rather the libraries didn't rely on environment variables... I think this will cause problems down the road if there are multiple users of the uvm library.

Reply:
OK, reverted that bit.